src/uct/tcp/tcp_cm.c

```diff
@@ -317,6 +325,44 @@ uct_tcp_ep_t *uct_tcp_cm_get_ep(uct_tcp_iface_t *iface,
     return ep;
 }
 
+uct_tcp_ep_t *uct_tcp_cm_get_conn_to_ep(uct_tcp_iface_t *iface,
```

Comment:
can this code be united with `uct_tcp_cm_get_ep` somehow?

Reply:
they are different